File: src-java/src/test/java/kigh/rmt/timer/TimerIdTest.java
package kigh.rmt.timer;

import org.junit.jupiter.api.Test;

import static org.assertj.core.api.Assertions.assertThat;
import static org.junit.jupiter.api.Assertions.*;

class TimerIdTest {
    @Test
    public void equals() {
        assertThat(new TimerId("hoge").equals(new TimerId("hoge"))).isTrue();
    }
}

File: src-kotlin/src/test/kotlin/kigh/rmt/timer/TimerIdTest.kt
package kigh.rmt.timer;

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test;

class TimerIdTest {
    @Test
    fun equals() {
        assertThat(TimerId("hoge").equals(TimerId("hoge"))).isTrue();
    }
}